src/physics/manufactured_solution.cpp

```diff
@@ -548,6 +576,37 @@ ::gradient (const dealii::Point<dim,real> &point, const unsigned int istate) con
     return gradient;
 }
 
+template <int dim, typename real>
+inline dealii::SymmetricTensor<2,dim,real> ManufacturedSolutionZero<dim,real>
+::hessian (const dealii::Point<dim,real> &/*point*/, const unsigned int /*istate*/) const
+{
+    dealii::SymmetricTensor<2,dim,real> hessian;
+    if (dim==1) {
```

Comment:
Same as above. Can be put in a for loop.

Reply:
I have also fixed this.